client/src/components/Notifications.js

```diff
@@ -45,70 +45,71 @@ const linethrough = {
 export default function NotificationsPopper(props) {
 	const classes = useStyles();
 
-	const cutProductName = (name) => {
-		try {
-			return name.slice(0, 1);
-		} catch {
-			return 'P';
-		}
-	};
-	console.log(props.messages);
-	return (
-		<div>
-			<Container align="center">
-				<Box className={classes.box}>
-					<ArrowDropUpRoundedIcon fontSize="large" className={classes.arrow} />
-				</Box>
-			</Container>
-			<Divider variant="fullWidth" component="li" className={classes.line} />
-			<List className={classes.list}>
-				{props.messages.reverse().map((msg) => (
-					<div key={msg.id} className={classes.listItem}>
-						<Typography color="textPrimary" variant="subtitle2">
-							{msg.type}
-						</Typography>
-						<ListItem component="a" href={msg.content.url} target="_blank" rel="noreferrer">
-							<ListItemAvatar>
-								<Avatar src={msg.content.image} variant="square" type="image">
-									{cutProductName(msg.content.title)}
-								</Avatar>
-							</ListItemAvatar>
-							<ListItemText>
-								<Typography className={classes.title} color="textPrimary" variant="subtitle2">
-									{cutContentLength(msg.content.title, 30, 'no product title')}
-								</Typography>
-								{msg.content.url ? (
-									<Typography variant="caption" color="textSecondary">
```

Comment:
Looking great! If we could keep these ternaries so my follower notifications don't break, I think we should be able to use the rest the same!

Reply:
Sorry, I didn't notice them. When I was merging, I missed these ternaries. I'm correcting those.